drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/ce_referral_task_export.rb

```diff
@@ -0,0 +1,71 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module HmisExternalApis::AcHmis::Exporters
+  class CeReferralTaskExport
+    include ::HmisExternalApis::AcHmis::Exporters::CsvExporter
+
+    def run!
+      Rails.logger.info 'Generating content of CE Referral Task export'
+
+      write_row(columns)
+      total = referral_tasks.count
+
+      Rails.logger.info "There are #{total} referral tasks to export"
+
+      referral_tasks.find_each.with_index do |task, i|
+        Rails.logger.info "Processed #{i} of #{total}" if (i % 1000).zero?
+        referral_id = instance_id_to_referral_id[task.instance_id]
+        raise "Referral not found for instance id '#{task.instance_id}'" unless referral_id
+
+        completed_by_hud_user_pk = to_hud_user_pk(task.updated_by) if task.updated_by
```

Comment:
```suggestion
        updated_by_hud_user_pk = to_hud_user_pk(task.updated_by) if task.updated_by
```
unless there is something I'm missing?